packages/react-core/src/components/Alert/examples/Alert.md

```diff
@@ -70,12 +93,142 @@ import { Alert, AlertActionCloseButton, AlertActionLink } from '@patternfly/reac
+        <Alert title="Default timeout Alert" timeout={timeout} actionLinks={
+          <React.Fragment>
+            <AlertActionLink>View details</AlertActionLink>
+            <AlertActionLink>Ignore</AlertActionLink>
+          </React.Fragment>
+        }>
+          This alert will dismiss after {`${timeout / 1000} seconds`}
+        </Alert>
+      ]
+    });
+  }
+
+  return (
+    <React.Fragment>
+      <Button variant="secondary" onClick={onClick}>Add alert</Button>
+      <Button variant="secondary" onClick={() => setAlerts([])}>Remove all alerts</Button>
+      <AlertGroup isLiveRegion>
+        {alerts}
+      </AlertGroup>
+    </React.Fragment>
+  );
+};
+```
+
+#### Expandable alerts
+
+An alert can contain additional, hidden information that is made visible by clicking a caret icon. This information can be expanded and collapsed each time the icon is clicked.
+
+It is not recommended to use an expandable alert using `timeout` within a toast [alert group](/components/alert#alert) because the alert could timeout before users have time to interact with and view the entire alert. See [alert accessibility considerations](https://www.patternfly.org/v4/components/alert/design-guidelines#accessibility-considerations) to understand the accessibility risks associated with using expandable alerts in toast alert groups. 
+
+```ts
+import React from 'react';
+import { Alert, AlertActionCloseButton, AlertActionLink } from '@patternfly/react-core';
+
+<React.Fragment>
+  <Alert
+    isExpandable
+    variant="success"
+    title="Success alert title"
+    actionClose={<AlertActionCloseButton onClose={() => alert('Clicked the close button')} />}
+    actionLinks={
+      <React.Fragment>
+        <AlertActionLink onClick={() => alert('Clicked on View details')}>View details</AlertActionLink>
+        <AlertActionLink onClick={() => alert('Clicked on Ignore')}>Ignore</AlertActionLink>
+      </React.Fragment>
+    }
+  >
+    <p>Success alert description. This should tell the user more information about the alert.</p>
+  </Alert>
+  <Alert
+    isExpandable
+    isInline
+    variant="success"
+    title="Success alert title"
+    actionClose={<AlertActionCloseButton onClose={() => alert('Clicked the close button')} />}
+    actionLinks={
+      <React.Fragment>
+        <AlertActionLink onClick={() => alert('Clicked on View details')}>View details</AlertActionLink>
+        <AlertActionLink onClick={() => alert('Clicked on Ignore')}>Ignore</AlertActionLink>
+      </React.Fragment>
+    } 
+  >
+    <p>Success alert description. This should tell the user more information about the alert.</p>
+  </Alert>
+</React.Fragment>
+```
+
+
+#### Truncated alerts
+
+Use the `truncateTitle` property to shorten a long alert message. Setting `truncateTitle` equal to a number (passed in as `{n}`) will reduce the number of lines of text in the alert's message. Users may hover over a truncated alert to see the full message in a popup. 
```

Comment:
Nitpick: maybe the verbiage could be updated to reflect that this property will only affect the alert title/the `title` property.